addons/mass_mailing/models/mailing.py

```diff
@@ -333,6 +347,53 @@ def _compute_render_model(self):
         for mailing in self:
             mailing.render_model = mailing.mailing_model_real
 
+    @api.depends('is_ab_testing', 'ab_testing_pc', 'campaign_id')
+    def _compute_ab_testing_description(self):
+        mailing_standard = self.filtered(lambda mailing: not mailing.is_ab_testing)
+        mailing_standard.ab_testing_description = ""
+        for mailing in self - mailing_standard:
+            message = ""
+            mailing_type_description = dict(self._fields.get('mailing_type').selection).get(mailing.mailing_type)
+            first_line = _(
+                "A sample of %(ab_testing_pc)s%% of recipients will receive this %(mailing_type)s",
+                ab_testing_pc=mailing.ab_testing_pc,
+                mailing_type=mailing_type_description,
+            )
+            if mailing.ab_testing_mailings_count >= 2:
+                if mailing.ab_testing_winner_selection == 'manual':
+                    message = _("Don't forget to send your prefered version")
+                else:
+                    winner_selection = mailing._get_winner_selection()
+                    message = _(
+                        "Then on %(schedule_date)s the %(mailing_type)s having the %(winner_selection)s will be sent",
+                        schedule_date=mailing.ab_testing_schedule_datetime.strftime("%b %d, %Y"),
+                        mailing_type=mailing_type_description,
+                        winner_selection=winner_selection,
+                    )
+            other_ab_testing_mailings = mailing._get_ab_testing_siblings_mailings().filtered(lambda m: m.is_ab_testing and m.id != mailing.id)
+            other_ab_testing_pc = sum(list(map(lambda m: m.ab_testing_pc, other_ab_testing_mailings)))
+            values = {
+                'ab_testing_mailings_count': mailing.ab_testing_mailings_count,
+                'ab_testing_done': mailing.campaign_id.ab_testing_done,
+                'ab_testing_pc': mailing.ab_testing_pc,
+                'first_line': first_line,
+                'mailing_type': mailing_type_description,
+                'other_ab_testing_pc': other_ab_testing_pc,
+                'other_ab_testing_mailing_count': mailing.ab_testing_mailings_count - 1,
+                'message': message,
+                'remaining_ab_testing_pc': 100 - (other_ab_testing_pc + mailing.ab_testing_pc),
+            }
+            mailing.ab_testing_description = self.env['ir.qweb']._render('mass_mailing.ab_testing_description', values)
+
+    @api.constrains('ab_testing_pc')
+    def _check_total_percentage_ab_test(self):
+        for campaign in self.campaign_id:
```

Comment:
Strange compute ... but completely not reveilled. Maybe @awa-odoo will have some insights ?